### PR TITLE
Prune some deps

### PR DESCRIPTION
**Pull request by eranrund**
### Motivation

Less dependencies = faster compilation time, less headache.

### In this PR
* Removing a bunch of unused dependencies

### CODE REVIEW

File: consensus/service/Cargo.toml [hunk 1]
```diff
@@ -26,18 +26,15 @@ mc-ledger-db = { path = "../../ledger/db" }
 mc-ledger-sync = { path = "../../ledger/sync" }
 mc-peers = { path = "../../peers" }
 mc-sgx-report-cache-untrusted = { path = "../../sgx/report-cache/untrusted" }
-mc-sgx-urts = { path = "../../sgx/urts" }
```
Comment (garbageslam) [inline consensus/service/Cargo.toml:29]: this might be needed when linking, i'm not sure

Comment (garbageslam) [reply consensus/service/Cargo.toml:29]: but actually, if so, it probably appears in the consensus/enclave proxy object's crate?

**Comment by eranrund** [comment]
I still need to go over this but I wanted it to go through CI.

**Review by garbageslam — APPROVED** [review]
thank you!

File: consensus/service/Cargo.toml (hunk 1 above)
Comment (eranrund) [reply consensus/service/Cargo.toml:29]: yeah, it's in the `consensus/enclave/Cargo.toml` file which I think is where it should be.

**Review by kylefleming — APPROVED** [review]
LGTM

**Pull request merged**